tests/phpunit/unit/api/utilities/beansGet.php

```diff
@@ -30,35 +30,90 @@ protected function setUp() {
 	}
 
 	/**
-	 * Test beans_get() should return the default value.
+	 * Test beans_get() should find the needle when array is given.
```

Comment:
I was a stickler for adding `an` or `the` in the last few PRs. 
So just for consistency (with myself and the last three tests), 
...when  **an** array is given.
and
...when **an** object is given.